accessibilityTest/AccessibilityOmnibox.ts

```diff
@@ -38,6 +40,67 @@ export const AccessibilityOmnibox = () => {
       done();
     });
 
+    describe('with keyboard navigation', () => {
+      let input: HTMLInputElement;
+
+      const triggerEvent = (key: KEYBOARD) => {
+        const event = new KeyboardEvent('keyup');
+        Object.defineProperty(event, 'keyCode', {
+          get: () => {
+            return key;
+          }
+        });
+        Object.defineProperty(event, 'which', {
+          get: () => {
+            return key;
+          }
+        });
+        input.dispatchEvent(event);
+      };
+
+      const down = () => triggerEvent(KEYBOARD.DOWN_ARROW);
+      const up = () => triggerEvent(KEYBOARD.UP_ARROW);
+
+      beforeEach(async done => {
+        const omniboxElement = getOmniboxElement();
+        getSearchSection().appendChild(omniboxElement.el);
+        await afterQuerySuccess();
+        showSuggestions(omniboxElement);
+        await afterDelay(500);
```

Comment:
Does `afterDelay` use `setTimeout`? If so, maybe you can get a faster performance with `jasmine.clock`?

Reply:
It's not a unit test, it's an integration test, so it really needs to wait 500ms, because it's actually executing a call to retrieves suggestions.